packages/donor/src/screens/authentication/resetpassword/ResetPasswordScreen.tsx

```diff
@@ -49,16 +50,18 @@ export default function ResetPasswordScreen(props: ResetPasswordScreenProps) {
         />
         <div className={styles.title}>איפוס סיסמה</div>
         <Input
+          name="reset_password"
           onChangeText={(emailContent) => {
             setEmail(emailContent);
             setError("");
           }}
           label={`דוא״ל`}
           value={email}
-          type="email"
+          type={InputType.Email}
```

Comment:
I'd say it has an intrinsic importance to the way I built Analytics,
as this enum easily categorizes types of inputs.
This would be more cumbersome not to implement imo

Reply:
In that case, you can first implement the Enum change in a PR, and the analytics change on top of it

Reply:
https://github.com/startach/ZM-BloodComponents/pull/377